docs/sources/alerting/notifications.md

```diff
@@ -111,6 +111,19 @@ Token | If provided, Grafana will upload the generated image via Slack's file.up
 
 If you are using the token for a slack bot, then you have to invite the bot to the channel you want to send notifications and add the channel to the recipient field.
 
+### Opsgenie
```

Comment:
This might be v8, not positive. @kylebrandt , can you confirm?

Reply:
yea, but only a part this (tags as extra properties, removed that)